R/release.R

```diff
@@ -152,29 +154,27 @@ release_type <- function(version) {
   }
 }
 
-#' Draft a GitHub release
+#' Publish a GitHub release
 #'
 #' @description
-#' Creates a __draft__ GitHub release for the current package. Once you are
-#' satisfied that it is correct, you will need to publish the release from
-#' GitHub. The key pieces of info are which commit / SHA to tag, the associated
-#' package version, and the relevant NEWS entries.
+#' Pushes the current branch (if safe) then publishes a GitHub release for the
+#' latest CRAN submission.
 #'
-#' If you use `devtools::release()` or `devtools::submit_cran()` to submit to
-#' CRAN, information about the submitted state is captured in a CRAN-SUBMISSION
-#' or CRAN-RELEASE file. `use_github_release()` uses this info to populate the
-#' draft GitHub release and, after success, deletes the CRAN-SUBMISSION or
-#' CRAN-RELEASE file.
-#'
-#' In the absence of such a file, we must fall back to assuming the current
-#' state (SHA of `HEAD`, package version, NEWS) is the submitted state.
+#' If you use [devtools::release()] or [devtools::submit_cran()] to submit to
+#' CRAN, information about the submitted state is captured in a
+#' `CRAN-SUBMISSION` or `CRAN-RELEASE` file. `use_github_release()` uses this
+#' info to populate the draft GitHub release and, after success, deletes the
+#' the file. In the absence of such a file, we assumes that current state
+#' (SHA of `HEAD`, package version, NEWS) is the submitted state.
 #'
 #' @param host,auth_token `r lifecycle::badge("deprecated")`: No longer
 #'   consulted now that usethis allows the gh package to lookup a token based on
 #'   a URL determined from the current project's GitHub remotes.
+#' @param draft If `TRUE`, publishes a draft release.
```

Comment:
```suggestion
#' @param publish If `TRUE`, publishes a release. If `FALSE`, creates a draft release.
```

It feels like the default and documentation flow more smoothly if we rig it the other way around.